Automattic/wp-calypso
# Help Center: add GM closure notice to Help Center

#### Please ignore the review request that was automatically sent to you. I changed the base branch and this pinged you by mistake.

#### Proposed Changes

* This adds a grand meetup closure notice to the Help Center. 
* I re-implemented the logic of [`gm-closure-notice`](https://github.com/Automattic/wp-calypso/blob/trunk/client/me/concierge/shared/gm-closure-notice.js#L33) just because I wanted to avoid pulling `moment` into the Help Center when there is no need for it.
* I reused all translated phrases and didn't make any changes to them.


<img width="432" alt="image" src="https://user-images.githubusercontent.com/17054134/198304863-2af4ff98-788c-4c7f-998f-d1660a8dabb5.png">


#### Testing Instructions

1. Change this [line](https://github.com/Automattic/wp-calypso/blob/add/gm-notice-to-help-center/packages/help-center/src/components/help-center-contact-page.tsx#L74) to make the date 27th of October. 
2. Open the Help Center with an account that has access to chat.
3. You should see the notice “Live chat support **will be** closed from Saturday, November 5th 01:00 AM until Monday, November 14th 08:00 AM. Customer support via email will remain open.”. Focus on “Will be”.
4. Change the same line to October 25th, and `closesAt` to October 27th, the message should be “Live chat support is closed until Monday, November 14th 08:00 AM. In the meantime you can still reach us by email.”. Focus on "Closed".
